packages/story-editor/src/components/canvas/selection.js

```diff
@@ -84,7 +84,9 @@ function Selection() {
   }
 
   // Multi-selection.
-  return <MultiSelectionMoveable selectedElements={selectedElements} />;
+  return (
+    <MultiSelectionMoveable {...props} selectedElements={selectedElements} />
+  );
```

Comment:
Just above here there is the `SingleSelectionMoveable` too. That component didn't need the `onContextMenu` prop since the `FrameLayer` handles the right click menu for single select.

💭 Probably should add a comment here about that, but still feels pretty weird.

Reply:
Would it be better just to just handle the `onContextMenu` in `SingleSelectionMoveable` instead of the `FrameLayer` to consolidate the approach? or would that cause regressions with the bg or some other behavior?

Reply:
Yeah so I tried adding `onContextMenu` to the single select but it wasn't working for some reason 🤔 wondering if the frame (when single selected) renders under the frame area.

Thinking about it more - I think the `onContextMenu` handler belongs here so it'd be better to make it work on `SingleSelectMovable`